docs/source/install.md

```diff
@@ -2,40 +2,32 @@
 
 ## Requirements
 
-* Singularity
+* Apptainer
 * Nextflow (`curl -s https://get.nextflow.io | bash`)
 
 **Recommended**
 
 * Conda
-* Singularity Remote Login
 
 ## Development deployment (self-contained)
 
 ### Copy code locally
 
 ```bash
 git clone --recurse-submodules --single-branch --branch master \\ 
-    git@github.com:genomic-medicine-sweden/jasen.git &&        \\
+    https://github.com/genomic-medicine-sweden/jasen.git &&        \\
```

Comment:
This creates a lot of problems with contributing down the line. I reckon we should leave it as is.

Reply:
Having the basic instruction naming HTTPS as a default over SSH makes the installation more accessible. 
In terms of security risk via rogue contributors, suggesting (not demanding) SSH kind of feels like putting unpowered security cameras in a barn.